test/exchange_wrapper_test.ts

```diff
@@ -767,16 +806,29 @@ describe('ExchangeWrapper', () => {
             const signedOrder = await fillScenarios.createFillableSignedOrderAsync(
                 makerTokenAddress, takerTokenAddress, makerAddress, takerAddress, fillableAmount,
             );
-            const orderHash = await zeroEx.getOrderHashHexAsync(signedOrder);
+            const orderHash = zeroEx.getOrderHashHex(signedOrder);
             const orderHashFromContract = await (zeroEx.exchange as any)
                 ._getOrderHashHexUsingContractCallAsync(signedOrder);
             expect(orderHash).to.equal(orderHashFromContract);
         });
     });
-    describe('#getContractAddressAsync', () => {
-        it('returns the exchange contract address', async () => {
-            const exchangeAddress = await zeroEx.exchange.getContractAddressAsync();
-            assert.isETHAddressHex('exchangeAddress', exchangeAddress);
+    describe('#getAvailableContractAddressedAsync', () => {
```

Comment:
`getAvailableContractAddressedAsync ` should be `getAvailableContractAddressesAsync `